docs/openapi.yaml

```diff
@@ -642,3 +691,85 @@ paths:
 
   # Index
   # "/graphql"
+
+  # Admin
+  '/ar-io/admin/debug':
+    get:
+      tags: [Admin]
+      summary: Admin debugging information.
+      description: Get detailed information about the current operational state of your AR.IO Gateway, including information about any current warnings or errors.
+      operationId: adminDebug
+      responses:
+        '200':
+          description: Successful operation.
+        '401':
+          description: Unauthorized, API key likely incorrect.
+      security:
+        - bearerAuth: []
+  '/ar-io/admin/queue-tx':
+    post:
+      tags: [Admin]
+      summary: Prioritize a transaction in the queue.
+      description: Stage a specific TX ID as priority for your Gateway to locate and index.
+      operationId: adminQueueTx
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+                  pattern: '^[0-9a-zA-Z_-]{43}$'
+                  description: TX ID of the transaction you would like to prioritize.
+            examples:
+              example1:
+                value:
+                  id: '3lyxgbgEvqNSvJrTX2J7CfRychUD5KClFhhVLyTPNCQ'
+      responses:
+        '200':
+          description: Successful operation
+        '401':
+          description: Unauthorized, API key likely incorrect.
+      security:
+        - bearerAuth: []
+  '/ar-io/admin/block-data':
+    put:
+      tags: [Admin]
+      summary: Blocks transactions or data-items so your AR.IO Gateway will not serve them.
+      description: | 
+        Submits a TX ID or data-item hash for content you do not want your AR.IO Gateway to serve. Once submitted, your Gateway will not respond to requests for these transactions or data-items.  
```

Comment:
`TX ID or data-item hash` should be `TX ID/data-item ID or sha-256 content hash`.